### ISSUE TITLE
Offre numérique : la redirection en cliquant sur "accéder à l'offre en ligne" ne fonctionne pas pour les liens commençant par www (marche avec https://)

### ISSUE THREAD

**Issue by abel-andre** [betagouv/pass-culture-browser#911]

**Comment by sixertoy** [comment]
Spotted sur `decouverte/AM3A/ALGA` thx
Si possible fourni une URL pour l'offre sur laquelle on peut reproduire le bug

**Comment by sixertoy** [comment]
Fixed pour la webapp
Testé avec l'offre `decouverte/AM3A/ALGA` -> `https://www.epresse.fr/`

@Ledoux @g4vroche 
Le fichier est ici
https://github.com/betagouv/pass-culture-browser/blob/master/src/helpers/parseOnlineOfferURL.js

Les tests sont là
https://github.com/betagouv/pass-culture-browser/blob/master/src/helpers/tests/parseOnlineOfferURL.spec.js

**Comment by g4vroche** [comment]
Comme dit dans cette issue : https://github.com/betagouv/pass-culture-browser/issues/921 , je pense que c'est à l'API de valider les URL et de fournir aux client des données valides.

Une chaîne de caractère ne commençant pas par le protocole n'est pas une URL valide.

**Comment by sixertoy** [comment]
True, ca a plus de sens de le mettre sur le pont entre les deux frontaux, plutot qu'éparpillé avec du JS
En python ca sera toujours plus stable
